contracts/integrations/examples/contractCreator/withTreasury/ResolverTaskCreator.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../../../OpsTaskCreator.sol";
+
+/**
+ * @dev
+ * Example contract that creates a resolver task.
+ */
+// solhint-disable not-rely-on-time
+// solhint-disable no-empty-blocks
+contract ResolverTaskCreator is OpsTaskCreator {
```

Comment:
I would rename it to `Counter` something to be more explicit about the example than the task argument
maybe `CounterResolverTaskCreator`

Reply:
[f773379](https://github.com/gelatodigital/ops/pull/88/commits/f7733796cdb371650c3c74b44893be4230616f63)